src/cas/pyodide-cas.ts

```diff
@@ -111,25 +111,87 @@ function usePythonConverter() {
     ['Sqrt', () => 'Sqrt'], // TODO: Replace with power
     ['Root', () => 'Root'], // TODO: Replace with power
     ['EqualEqual', () => 'sympy.Eq'],
+    ['Parentheses', () => ''],
+
+    ['Abs', () => 'sympy.Abs'],
+
+    ['Sin', () => 'sympy.sin'],
+    ['Cos', () => 'sympy.cos'],
+    ['Tan', () => 'sympy.tan'],
+
+    ['Sec', () => 'sympy.sec'],
+    ['Csc', () => 'sympy.csc'],
+    ['Cot', () => 'sympy.cot'],
+
+    ['Arcsin', () => 'sympy.asin'],
+    ['Arccos', () => 'sympy.acos'],
+    ['Arctan', () => 'sympy.atan'],
+
+    ['Asec', () => 'sympy.asec'],
+    ['Acsc', () => 'sympy.acsc'],
+    ['Atan', () => 'sympy.atan'],
+
+    ['Sinh', () => 'sympy.sinh'],
+    ['Cosh', () => 'sympy.cosh'],
+    ['Tanh', () => 'sympy.tanh'],
+
+    ['Sech', () => 'sympy.sech'],
+    ['Csch', () => 'sympy.csch'],
+    ['Coth', () => 'sympy.coth'],
+
+    ['Arsinh', () => 'sympy.asinh'],
+    ['Arcosh', () => 'sympy.acosh'],
+    ['Artanh', () => 'sympy.atanh'],
+
+    ['Asech', () => 'sympy.asech'],
+    ['Acsch', () => 'sympy.acsch'],
+    ['Acoth', () => 'sympy.acoth'],
+
+    ['Pi', () => 'sympy.pi'],
+    ['ImaginaryUnit', () => 'sympy.I'], // TODO: test
+    ['ExponentialE', () => 'sympy.E'], // TODO: test
+    ['GoldenRatio', () => 'sympy.GoldenRatio'], // TODO: test
+    ['EulerGamma', () => 'sympy.EulerGamma'], // TODO: test
+
+    // TODO: MathLive latex serial/parse destroys ln,log, etc? in ExpressionElement.vue line 78
+    // ['log', () => 'sympy.log'],
   ])
 
+  const KnownLatexFunctions = {
+    '\\sin': 'sympy.sin',
+  }
```

Comment:
This is definitely one of those functions that the compute-engine is supposed to support but currently doesn't. To be fair, the compute-engine is super new...

Reply:
Also, I think the neatest place to add functions would be in the mathlive/compute-engine [dictionary](https://cortexjs.io/compute-engine/guides/dictionaries/)

Reply:
That would be great, I just spent some time attempting to do just that, so I am curious as to how you would do it. I think adding `\\solve` `\\factor`, and `\\expand` to the dictionary and the content inside of the custom EVALUATE `\\xrightarrow` you added were parsed as MathJSON would be cleaner.